v8.go

```diff
@@ -184,9 +200,11 @@ type Context struct {
 	ptr C.ContextPtr
 
 	callbacks      map[int]callbackInfo
+	callbacksMutex sync.RWMutex
```

Comment:
I did this a while ago, I can't remember at the time why, the way I am using this in my projects it shouldn't be accessed by another goroutine. I will try to reproduce the issue I was running into and create a test for it if I can.